core/trino-main/src/main/java/io/trino/SystemSessionProperties.java

```diff
@@ -311,6 +313,11 @@ public SystemSessionProperties(
                         "Adapt plan to pre-partitioned tables",
                         featuresConfig.isPlanWithTableNodePartitioning(),
                         false),
+                doubleProperty(
+                        TABLE_SCAN_PARTITIONING_MIN_BUCKET_TO_TASK_RATIO,
+                        "Min table scan bucket to task ratio in order to adapt plan to pre-partitioned tables",
```

Comment:
`Min table scan bucket to task ratio for which plan will be adopted to pre-partitioned tables`